style.css

```diff
@@ -0,0 +1,168 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap");
+
+:root {
+  --black: #1c1a19;
+  --dark-gray: #3c3a39;
+  --orange: #ff6b00;
+  --light: #fff5e1;
+}
+
+body {
+  background-color: var(--black);
+  margin: 0;
+  width: 100%;
+  min-width: 395px;
+}
+
+.appBar {
+  height: 17vh;
+  background-color: var(--dark-gray);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.appBar a {
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+}
+
+#menu-icon {
+  width: 24px;
+  right: 25px;
+  margin-right: 25px;
+  margin-bottom: 25px;
+}
+
+.logo-text {
+  width: 250px;
+  font-size: 3.5vh;
+  color: white;
+  margin-left: 8%;
+  margin-bottom: 20px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+}
+
+.section-headline {
+  height: 100vh;
+  margin-left: 24px;
+  background-image: url("images/Group.png");
+  background-repeat: no-repeat;
+  background-position: right bottom;
+}
+
+.heading {
+  /* Primary Text */
+  height: 124px;
+  margin-top: 120px;
+  color: var(--orange);
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10vh;
+  line-height: 64px;
+
+  /* or 114% */
+
+  letter-spacing: 0.37px;
+  margin-bottom: 0;
+}
+
+.sub-heading {
+  color: var(--light);
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 5vh;
+  line-height: 32px;
+
+  /* or 133% */
+}
+
+.description {
+  height: 123px;
+  margin-top: 12px;
+  color: white;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.section-social-media {
+  margin-left: 5px;
+  padding: 0;
+}
+
+.links {
+  margin: 0;
+  padding-left: 0;
+  display: flex;
+  list-style: none;
+}
+
+.links li {
+  margin: 0;
+}
+
+.links li a {
+  margin-right: 30px;
+}
+
+@media (max-width: 395px) {
+  body {
+    max-width: 395px;
+  }
+
+  .appBar {
+    height: 92px;
+  }
+
+  #menu-icon {
+    width: 18px;
+    right: 23px;
+    margin-right: 20px;
+    margin-bottom: 20px;
+  }
+
+  .logo-text {
+    font-size: 18px;
+    margin-left: 12px;
+    margin-bottom: 20px;
+  }
+
+  .heading {
+    font-size: 56px;
+    line-height: 64px;
+    width: 327px;
+  }
+
+  .sub-heading {
+    margin-top: 12px;
+    height: 27px;
+    font-weight: 700;
+
+    /* or 133% */
+  }
+
+  .section-headline {
+    background-image: url("images/Group.png");
+    background-repeat: no-repeat;
+    background-position: 100% 110%;
+  }
```

Comment:
- [ ] The background image is located a little lower than in the original design, please make sure to place it upper. 

Your design  | Figma design
------------- | -------------
<img width="380" alt="Screenshot 2022-03-18 at 15 49 01" src="https://user-images.githubusercontent.com/77038610/158997930-0a3232e6-970e-4814-8242-ca4ef1e009b0.png">   |  <img width="362" alt="Screenshot 2022-03-18 at 15 48 09" src="https://user-images.githubusercontent.com/77038610/158997841-106fa684-a3be-4da1-ae10-2b201f48b333.png">

Reply:
Fixed through adjusting the height of class .headline-section(#52 in style.css) to 590px exact pixel to the figma template

Reply:
- Nav tag is nested in header tag. Height is also adjusted as per recommendation
- Elements in nav menu are aligned with flexbox. 
- Social media icons are horizontally alligned using flexbox
- Background image is matched to the one in the template